### PR TITLE
Add and 'implement' "listByronTransactions" to the API

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

#779

# Overview

- [x] I have added a `GET /byron-wallets/{walletId}/transactions` to get the list of transactions for a byron wallet
- [x] The implementation is actually the same as the one for sequential wallets, so we could consider testing done here :sweat_smile: ? @piotr-iohk ?

# Comments

### CODE REVIEW

File: specifications/api/swagger.yaml [hunk 1]
```diff
@@ -1248,6 +1249,17 @@ paths:
...
 
+  /byron-wallets/{walletId}/transactions:
+    get:
+      operationId: listByronTransactions
+      tags: ["Byron Transactions"]
+      summary: List
+      description: |
+        <p align="right">status: <strong>under testing</strong></p>
+
+        List all incoming and outgoing wallet's transactions.
```
Comment (jonathanknowles) [inline specifications/api/swagger.yaml]: This sentence seems a bit strange. I would write it as this:

```suggestion
        List all incoming and outgoing transactions for the given wallet.
```

**Review by jonathanknowles — DISMISSED** [review]
This looks good, aside from the plain-English description of the endpoint in the API specification, which I recommend revising.

File: specifications/api/swagger.yaml (hunk 1 above)
Comment (jonathanknowles) [reply specifications/api/swagger.yaml]: I would add that the reason it's strange is that it parses as "transactions of incoming wallets and outgoing wallets", when in fact what we want is "incoming transactions and outgoing transactions of a wallet".

**Comment by piotr-iohk** [comment]
> The implementation is actually the same as the one for sequential wallets, so we could consider testing done here sweat_smile ? @piotr-iohk ?

I don't think so! :)
Although I don't think we'd need to copy existing tests 1-1. Especially that we cannot do transactions on byron wallets...

File: specifications/api/swagger.yaml [hunk 2]
```diff
@@ -1285,3 +1291,27 @@ paths:
...
+  /network/information:
+    get:
+        operationId: getNetworkInformation
+        tags: ["Network"]
+        summary: Information
+        description: |
+          <p align="right">status: <strong>stable</strong></p>
+        responses: *responsesGetNetworkInformation
+
+  /proxy/transactions:
```
Comment (rvl) [inline specifications/api/swagger.yaml]: I'm feeling lukewarm about the name "proxy" (conveys mechanism), but this is probably splitting hairs.

**Review by rvl — APPROVED** [review]
LGTM

File: specifications/api/swagger.yaml (hunk 1 above)
Comment (KtorZ) [reply specifications/api/swagger.yaml]: Done :+1:

**Pull request merged**